oak_loader/src/options.rs

```diff
@@ -348,11 +343,8 @@ fn get_oidc_client_info(opt: &Opt) -> anyhow::Result<Option<ClientInfo>> {
 }
 
 /// Gets the default root TLS certificates from the embedded byte array.
-fn get_default_root_tls_certs() -> anyhow::Result<String> {
-    let result = std::str::from_utf8(include_bytes!("certs/roots.pem"))
-        .context("could not read embedded PEM-encoded root TLS certificates as a UTF8 string")?
-        .to_owned();
-    Ok(result)
+fn get_default_root_tls_certs() -> Vec<u8> {
+    include_bytes!("certs/roots.pem").to_vec()
```

Comment:
The `TLS certificate` fields in the configurations are now `Vec<u8>`. So I changed it here too. I am not sure how important the check for conversion to a UTF8 string is. But this change list moves all such checks into the `GrpcClientNode::new` (basically inside `load_certificate`). Do we generally want to have these checks earlier, or is it ok to postpone them until the actual node creation?

Reply:
Added a call to `rustls::internal::pemfile::certs`, just for checking the validity of the certificate. But the resulting certificate object is discarded. The same check happens inside `load_certificate`, which I think can be left there as well, since that function might be called independently too.